<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UFC_UGC_ZOS_window</class>
 <widget class="QMainWindow" name="UFC_UGC_ZOS_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1178</width>
      <height>771</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,3,1">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="state_label">
         <property name="text">
          <string>未启动</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="styleSheet">
          <string notr="true">QLabel{
color:red;
}</string>
         </property>
         <property name="text">
          <string>串口配置：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="port_combox">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="refresh_port_btn">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>刷新串口</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="start_btn">
         <property name="text">
          <string>启动</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="run_btn">
         <property name="text">
          <string>运行</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stop_btn">
         <property name="text">
          <string>停机</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="open_log_btn">
         <property name="text">
          <string>打开日志文件夹</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="auto_btn">
         <property name="text">
          <string>自动运行流程</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="disabled_auto_btn">
         <property name="text">
          <string>停止自动运行流程</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>响应信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="responselist"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
